test/squish/suite_VerifySettingsDialog/shared/scripts/gui_helpers.py

```diff
@@ -0,0 +1,166 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# -*- coding: utf-8 -*
+# mypy: disable-error-code="attr-defined"
+
+import gui_locators
+import squish
+import test
+
+snoozeTimeout = 3  # seconds
+
+
+def launchDeadlineConfigGUI():
+    squish.startApplication("deadline config gui")
+    test.log("Launching Deadline Config GUI")
+    test.log(
+        "Sleep for " + str(snoozeTimeout) + " seconds to allow GUI authentication to fully load."
+    )
+    squish.snooze(snoozeTimeout)
+    test.compare(
+        squish.waitForObjectExists(
+            gui_locators.aWS_Deadline_Cloud_workstation_configuration_DeadlineConfigDialog
+        ).visible,
+        True,
+        "Expect the Deadline Config GUI to be open.",
+    )
+
+
+def setGlobalSettings(profileName: str, jobHistDir: str):
+    test.compare(
+        str(squish.waitForObjectExists(gui_locators.profile_settings_QLineEdit).displayText),
+        jobHistDir,
+        "Expect job history directory default path to be equal.",
+    )
+    squish.mouseClick(
+        squish.waitForObjectItem(gui_locators.global_settings_AWS_profile_QComboBox, profileName),
+        484,
+        9,
+        squish.Qt.NoModifier,
+        squish.Qt.LeftButton,
+    )
+    squish.mouseClick(
+        squish.waitForObjectItem(gui_locators.global_settings_AWS_profile_QComboBox, profileName),
+        225,
+        6,
+        squish.Qt.NoModifier,
+        squish.Qt.LeftButton,
+    )
+    test.log("Set Global settings")
+
+
+def setProfileSettings(farmName: str):
+    squish.mouseClick(
+        squish.waitForObject(gui_locators.profile_settings_QComboBox),
+        407,
+        13,
+        squish.Qt.NoModifier,
+        squish.Qt.LeftButton,
+    )
+    squish.mouseClick(
+        squish.waitForObjectItem(gui_locators.profile_settings_QComboBox, farmName),
+        189,
+        10,
+        squish.Qt.NoModifier,
+        squish.Qt.LeftButton,
+    )
+    test.log("Set Profile settings")
+
+
+def setFarmSettings(queueName: str, storageProfile: str, jobAttachments: str):
+    squish.mouseClick(
+        squish.waitForObject(gui_locators.farm_settings_QComboBox),
```

Comment:
Can we add some comments for mouse clicks on what is being tested? Makes it a bit easier to parse. Some of the locator names are not very helpful.

Reply:
The locator names are pre-set by Squish but they can be changed so I'll look into it. Will look into adding comments regarding the mouse-clicks as well. Thanks!